src/index.ts

```diff
@@ -240,16 +239,23 @@ export function isSubset (set: Uint8Array[], maybeSubset: Uint8Array[]): boolean
   return (intersection.length === maybeSubset.length)
 }
 
-export interface WebTransportConfig {
-  maxInboundStreams: 1000
```

Comment:
By specifying this as `1000` and not `number` the user can only ever pass `1000` in which I don't think was the intention.